### PR TITLE
docs: blob hub introduction and guideline for developers

### PR DESCRIPTION
**Pull request by alexgao001**

### CODE REVIEW

**Review by huihzhao — COMMENTED** [review]
We should use BEP336, instead of EIP4844. Use the same terminology

File: docs/bnb-greenfield/for-developers/blob-hub.md [hunk 1]
```diff
@@ -0,0 +1,384 @@
...
+decentralized technology in the field of data ownership and the data economy. Greenfield guarantees data availability,
+integrity, and accessibility through decentralized redundant storage and "Proof-of-Challenge", this makes it an ideal
+solution for a data archive layer.
+
+# BNB Greenfield as Data Archive Layer
+
+The Greenfield community has recently launched "BlobHub," a data archive layer designed for all layer 2 blockchains and
+Dapps which leverages EIP4844 blobs as data availability layer. All historical blobs can be persisted into Greenfield,
+and users can easily access these blobs whenever they want to query them. BlobHub not only serves Ethereum but also
+other blockchain that enable EIP4844.
```
Comment (huihzhao) [inline docs/bnb-greenfield/for-developers/blob-hub.md:35]: Blob-carrying transactions

File: mkdocs.yml [hunk 2]
```diff
@@ -239,6 +239,7 @@ nav:
               - JS SDK Official Doc: https://docs.bnbchain.org/greenfield-js-sdk/
           - How to Contribute: ./bnb-greenfield/for-developers/contribute.md
 
+          - Blob Hub: ./bnb-greenfield/for-developers/blob-hub.md
```
Comment (1nanosecond) [inline mkdocs.yml:242]: I think the order is also important, `How to Contribute` is always the last one,  `bundle service` and `blob hub` is close related, they should be the bottom ones.

File: docs/bnb-greenfield/for-developers/blob-hub.md [hunk 3]
```diff
@@ -0,0 +1,384 @@
+---
+title: Blob Hub
+description: Build “Blob” Data Archive on Greenfield
+keywords: [ BNB Greenfield, NodeReal, Blob hub ]
+---
+
+# Blob Hub
+
+# What Is Data Archive Layer?
```
Comment (1nanosecond) [inline docs/bnb-greenfield/for-developers/blob-hub.md:9]: I think we should split this page into two page: "Data Archive Layer" and `Blobhub`, as blobhub is an example of `Build on Data Archive Layer`

File: docs/bnb-greenfield/for-developers/blob-hub.md [hunk 4]
```diff
@@ -0,0 +1,384 @@
...
+
+This layer essentially maintains the integrity and trust of the blockchain, allowing everyone to independently verify
+the validity of transactions. The data availability layer guarantees access to newly created data, but it does not
+provide access to the entire historical data. For example, EIP4844 and Celestica will discard blob data older than 18
+days.
+
+The data archive layer is an extension of the data availability layer, ensuring that all historical block data remains
+publicly accessible after being archived.
+
+# What is BNB Greenfield
```
Comment (1nanosecond) [inline docs/bnb-greenfield/for-developers/blob-hub.md:23]: Please delete `What is BNB Greenfield` section here.

File: docs/bnb-greenfield/for-developers/blob-hub.md [hunk 5]
```diff
@@ -0,0 +1,384 @@
...
+Or you can use the script, replace `GRANTEE_BUNDLE_ACCOUNT` with the addr got from Bundle Service in step 2, and modify the `ALLOWANCE` to your expect amount, so that transacion
+gas will be paid your account:
+
+```shell
+bash scripts/set_up.sh --grant
+```
+
+After above steps are done, you can start running the Blob Syncer Service.
+
+### Build the binaries
```
Comment (1nanosecond) [inline docs/bnb-greenfield/for-developers/blob-hub.md:248]: This part is not necessary, user are not coming to build the blob hub

File: mkdocs.yml (hunk 2 above)
Comment (alexgao001) [reply mkdocs.yml:242]: moved

File: docs/bnb-greenfield/for-developers/blob-hub.md (hunk 3 above)
Comment (alexgao001) [reply docs/bnb-greenfield/for-developers/blob-hub.md]: splited

File: docs/bnb-greenfield/for-developers/blob-hub.md (hunk 4 above)
Comment (alexgao001) [reply docs/bnb-greenfield/for-developers/blob-hub.md]: deleted

File: docs/bnb-greenfield/for-developers/blob-hub.md [hunk 6]
```diff
@@ -0,0 +1,384 @@
...
+
+![Blob Hub Solution](../static/asset/blob-hub-1.png)
+Blob Hub Solution
+
+The syncing process ensures that no blob is missing and each blob synced to Greenfield is consistent. This is achieved
+by running a post-verification process that scans all uploaded blobs again, conducts integrity checks against data
+already stored in Greenfield, and detects any missing data. Duplicate uploads are prevented by key naming constraints in
+the Bundle service and Greenfield.
+
+## Support Network
```
[the remark this answers is not in the document]
Comment (alexgao001) [reply docs/bnb-greenfield/for-developers/blob-hub.md]: moved the endpoints under Greenfield endpoints page, the API spec will refer the blobhub repo readme

File: docs/bnb-greenfield/for-developers/blob-hub.md (hunk 5 above)
Comment (alexgao001) [reply docs/bnb-greenfield/for-developers/blob-hub.md]: only shown in the blobhub repo readme

**Review by unclezoro — APPROVED** [review]

**Pull request merged**